qutip/core/environment.py

```diff
@@ -1114,12 +1114,12 @@ def approx_by_pade(
     def _pade_params(self, Nk):
         eta_p, gamma_p = self._corr(Nk)
 
-        ck_real = [np.real(eta) for eta in eta_p]
+        ck_real = [eta for eta in eta_p]
         vk_real = [gam for gam in gamma_p]
```

Comment:
We could simplify this a bit

```suggestion
        ck_real, vk_real = self._corr(Nk)
```